doc/source/reference/c-api.types-and-structures.rst

```diff
@@ -377,13 +380,28 @@ PyArrayDescr_Type
     normally a Python string. These tuples are placed in this
     dictionary keyed by name (and also title if given).
 
+.. c:member:: PyObject *PyArray_Descr.names
+
+    A table of functions specifiv for each
```

Comment:
This isn't the right description. I think it should be the one from the comment *above* (not below) the field, the one about "An ordered tuple of field names or NULL if no fields are defined"